docs/source/conf.py

```diff
@@ -100,7 +100,7 @@
 intersphinx_mapping = {
     "python": ("https://docs.python.org/3", None),
     "numpy": ("https://numpy.org/doc/stable/", None),
-    "pygfx": ("https://pygfx.com/latest", None),
+    "pygfx": ("https://pygfx.com/stable", None),
```

Comment:
we should link to latest

Reply:
before when I had it at latest it failed to build